{{cookiecutter.repo_name}}/docs/getting_started.rst

```diff
@@ -22,51 +22,38 @@ When developing locally, it may be useful to have settings overrides that you do
-+-----------------------------------------------------+----------------------------------------------------------------------------+--------------------------------------------------------------------------+
-| SOCIAL_AUTH_EDX_OIDC_SECRET                         | OAuth 2.0 client secret                                                    | (This should be set to the value generated when the client was created.) |
-+-----------------------------------------------------+----------------------------------------------------------------------------+--------------------------------------------------------------------------+
-| SOCIAL_AUTH_EDX_OIDC_URL_ROOT                       | OAuth 2.0 authentication URL                                               | http://127.0.0.1:8000/oauth2                                             |
-+-----------------------------------------------------+----------------------------------------------------------------------------+--------------------------------------------------------------------------+
-| SOCIAL_AUTH_EDX_OIDC_ID_TOKEN_DECRYPTION_KEY        | OIDC ID token decryption key. This value is used to validate the ID token. | (This should be the same value as SOCIAL_AUTH_EDX_OIDC_SECRET.)          |
-+-----------------------------------------------------+----------------------------------------------------------------------------+--------------------------------------------------------------------------+
++-----------------------------------+----------------------------------+--------------------------------------------------------------------------+
+| Setting                           | Description                      | Value                                                                    |
++===================================+==================================+==========================================================================+
+| SOCIAL_AUTH_EDX_OAUTH2_KEY        | OAuth 2.0 client key             | (This should be set to the value generated when the client was created.) |
++-----------------------------------+----------------------------------+--------------------------------------------------------------------------+
+| SOCIAL_AUTH_EDX_OAUTH2_SECRET     | OAuth 2.0 client secret          | (This should be set to the value generated when the client was created.) |
++-----------------------------------+----------------------------------+--------------------------------------------------------------------------+
+| SOCIAL_AUTH_EDX_OAUTH2_URL_ROOT   | OAuth 2.0 authentication URL     | http://127.0.0.1:18000/oauth2                                            |
++-----------------------------------+----------------------------------+--------------------------------------------------------------------------+
```

Comment:
Shall we add the Backend-specific  settings as well?  `BACKEND_SERVICE_EDX_OAUTH2_KEY` and `BACKEND_SERVICE_EDX_OAUTH2_SECRET`?

Reply:
Yes - definitely! I will add them...

Reply:
Added.